src/components/common/calendar/index.tsx

```diff
@@ -32,31 +32,31 @@ export const Calendar = ({
         day: cn(
           'text-white/60 h-32 w-36 p-0 rounded-8 border-box',
           'focus-visible:border focus-visible:border-white/50 focus-visible:outline-none',
-          'hover:bg-background-900 hover:text-white'
+          'hover:!bg-background-900 hover:text-white'
```

Comment:
Why do you need !important now ?

Reply:
Cannot control how some tailwind classes are merged here, and updating to tailwind showed some differences to production